config/samples/backup.yaml

```diff
@@ -53,7 +53,7 @@ spec:
         secret:
           secretName: fdb-kubernetes-operator-secrets
   snapshotPeriodSeconds: 3600
-  version: 6.3.22
+  version: 7.1.26
```

Comment:
Why "7.1.26" here (instead of "7.1.27")?

Reply:
I chose 7.1.26 as this version has no AVX support (otherwise it could result in some errors that are not obvious if you don't know that AVX support is required).